determined-ai/determined
# chore: UI fix and improvements

## Description
- [DET-7394]
- Add sign-in animation



## Test Plan
- Check if new project/workspace/model button won't remain highlighted after closing modal
- Check if User settings dropdown won't remain   highlighted after clicking
- Check if there is a spin after clicking sign in button



## Commentary (optional)




## Checklist

See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[DET-7394]: https://determinedai.atlassian.net/browse/DET-7394?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
